### ISSUE TITLE
Delete or hide already sent campaigns

### ISSUE THREAD

**Issue by MathiasOliveira** [mettle/sendportal#81]
It would be nice if we were able to remove already sent campaigns from the dashboard

**Comment by Sepremex** [comment]
Yup, I was looking for this too, may be move it to the an archive campaigns section, if the campaigns has been sent then move it from the campaigns section,

**Comment by michaelzap** [comment]
That tab feature is great, but it would also be very helpful to be able to delete old campaigns (I have a number of tests, for example), as well as other elements (subscribers, templates, etc.). I imagine that there are consistency issues involved so that links among elements don't break, but it would be worth resolving these to make it possible to delete things related to sent campaigns. At least in my use case, it would allow me to clean up cruft (and potentially honor requests for deletion of personal information as needed).
